### PR TITLE
Refresh the repo README

### PR DESCRIPTION
**Pull request by rileykarson**
Parallel to https://github.com/hashicorp/terraform-provider-google/pull/10481

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,52 @@
...
+- Tutorials: [learn.hashicorp.com](https://learn.hashicorp.com/terraform?track=getting-started#getting-started)
+- Forum: [discuss.hashicorp.com](https://discuss.hashicorp.com/c/terraform-providers/tf-google/)
 - Documentation: https://www.terraform.io/docs/providers/google/index.html
 - [![Gitter chat](https://badges.gitter.im/hashicorp-terraform/Lobby.png)](https://gitter.im/hashicorp-terraform/Lobby)
 - Mailing list: [Google Groups](http://groups.google.com/group/terraform-tool)
-<img src="https://cdn.rawgit.com/hashicorp/terraform-website/master/content/source/assets/images/logo-hashicorp.svg" width="600px">
 
-Maintainers
------------
+The Terraform Google provider is a plugin for Terraform that allows for management of GCP resources.
```
Comment (megan07) [inline README.md:11]: Do we want to add `Beta` in here? We leave it out in a few places throughout the doc, I have no strong preference, just wanted to mention it.

File: README.md [hunk 2]
```diff
@@ -1,24 +1,52 @@
...
+terraform init -upgrade
+```
 
-Requirements
-------------
+to upgrade to the latest stable version of the Google provider. See the [Terraform website](https://www.terraform.io/docs/configuration/providers.html#provider-versions)
+for more information on provider upgrades, and how to set version constraints on your provider.
 
-- [Terraform](https://www.terraform.io/downloads.html) 0.12+
+Using the provider
```
Comment (megan07) [inline README.md:41]: These 2 sections seemed to be re-written to what you have above in TPG (`Documentation` and `Upgrading the provider`), I'm guessing you meant to remove them completely here?

**Review by megan07 — APPROVED** [review]
Thanks!

**Comment by raphaelauv** [comment]
why not merge ?

**Comment by megan07** [comment]
Closing due to the [more recent change](https://github.com/hashicorp/terraform-provider-google-beta/pull/4678)

**Pull request closed without merging**